x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/dot_expander.tsx

```diff
@@ -40,11 +40,15 @@ export const DotExpander: FunctionComponent = () => {
           {
             validator: ({ value }) => {
               if (typeof value === 'string' && value.length) {
-                return !value.includes('.')
+                const allowedPattern = value.includes('.') || value === '*';
+                return !allowedPattern
                   ? {
                       message: i18n.translate(
                         'xpack.ingestPipelines.pipelineEditor.dotExpanderForm.fieldNameRequiresDotError',
-                        { defaultMessage: 'A field value requires at least one dot character.' }
+                        {
+                          defaultMessage:
+                            'The field value should contain at least one dot character or equal to an asterisk.',
```

Comment:
I've reached out to our docs team to review this copy change.

Reply:
Thanks for working on this @shanwong29. I think we can shorten this as follows:

```suggestion
                            'The field name must be an asterisk or contain a dot character.',
```

What do you think?